<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>通信方式</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="radioRS232">
           <property name="text">
            <string>RS-232</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioUDP">
           <property name="text">
            <string>UDP</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioTCPServer">
           <property name="text">
            <string>TCP (Server)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioTCPClient">
           <property name="text">
            <string>TCP (Client)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>接口配置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>串行端口</string>
           </property>
           <property name="buddy">
            <cstring>boxSPort</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="boxSPort"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>波特率</string>
           </property>
           <property name="buddy">
            <cstring>boxBaudRate</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QComboBox" name="boxBaudRate"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>网络设备</string>
           </property>
           <property name="buddy">
            <cstring>boxNetDevice</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="boxNetDevice"/>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>数据位</string>
           </property>
           <property name="buddy">
            <cstring>boxDataBits</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QComboBox" name="boxDataBits"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>通信端口</string>
           </property>
           <property name="buddy">
            <cstring>editPortNum</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="editPortNum"/>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>停止位</string>
           </property>
           <property name="buddy">
            <cstring>boxStopBits</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QComboBox" name="boxStopBits"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>目标地址</string>
           </property>
           <property name="buddy">
            <cstring>editDestAddress</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="editDestAddress"/>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>奇偶校验</string>
           </property>
           <property name="buddy">
            <cstring>boxParity</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QComboBox" name="boxParity"/>
         </item>
        </layout>
        <zorder>label</zorder>
        <zorder>boxSPort</zorder>
        <zorder>label_2</zorder>
        <zorder>boxNetDevice</zorder>
        <zorder>label_3</zorder>
        <zorder>editPortNum</zorder>
        <zorder>label_4</zorder>
        <zorder>editDestAddress</zorder>
        <zorder>label_5</zorder>
        <zorder>boxBaudRate</zorder>
        <zorder>label_6</zorder>
        <zorder>boxDataBits</zorder>
        <zorder>label_7</zorder>
        <zorder>boxStopBits</zorder>
        <zorder>boxParity</zorder>
        <zorder>label_8</zorder>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>描述信息</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QTextBrowser" name="textBrowser"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btnScan">
          <property name="text">
           <string>扫描新设备</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnApply">
          <property name="text">
           <string>应用</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnClose">
          <property name="text">
           <string>关闭</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>671</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>btnClose</tabstop>
  <tabstop>radioRS232</tabstop>
  <tabstop>radioUDP</tabstop>
  <tabstop>radioTCPServer</tabstop>
  <tabstop>radioTCPClient</tabstop>
  <tabstop>boxSPort</tabstop>
  <tabstop>boxBaudRate</tabstop>
  <tabstop>boxDataBits</tabstop>
  <tabstop>boxStopBits</tabstop>
  <tabstop>boxParity</tabstop>
  <tabstop>boxNetDevice</tabstop>
  <tabstop>editPortNum</tabstop>
  <tabstop>editDestAddress</tabstop>
  <tabstop>btnScan</tabstop>
  <tabstop>btnApply</tabstop>
  <tabstop>textBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>618</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>618</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
